extension/content_script.js

```diff
@@ -21,6 +21,7 @@ var speed_bias = 500; // Minimum amount of speed spent on each sentence (in mill
 var speed_adj = 0; // Speed after it has been adjusted by sentence length
 // If the screen is currently scrolling. If it is, pause the tracker.
 var isScrolling = false;
+var currentStyle = "markedTwo"; // TEMPORARY global variable, just for style experimentation. Will get rid of later
```

Comment:
I prefer it being a variable like this actually.
Ez to change.
Someday if want to provide customization, this variable will be used too

Reply:
Nothing to do here, just resolve